website/src/content/docs/pt-br/index.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Biome
+head:
+  - tag: title
+    content: Biome
+template: splash
+description: Formate, verifique erros e muito mais em uma fração de segundo.
+hero:
+  title: Uma toolchain para seu projeto web
```

Comment:
Suggestion: translate the "toolchain" to "conjunto de ferramentas"

Reply:
Implemented suggestion in commit 7ee1d34